### PR TITLE
Fixes #26242 - Working message on custom cron interval

### PR DESCRIPTION
**Pull request by sjha4**
Depends on https://github.com/Katello/bastion/pull/243
- Create a sync plan (Hourly)
- Go to edit page of sync plan and select custom cron as interval
- You will see the working spinner and the text "working "and the cron text field will become visible.
- The page waits for the user to enter cron logic and save
- We would want to change the text "working" to something more clear, like "Please enter cron below"

### CODE REVIEW

**Comment by jturel** [comment]
I'm wondering if we need any text to show at all when using a custom cron interval. Do you think we need it / is it feasible to disable it?

**Comment by sjha4** [comment]
We can remove the text..It seems like a helpful message though to let users know? It used to say working earlier which is the root issue here..

**Comment by jturel** [comment]
Ah ok! If this is a regression I'm OK with fixing it. I'll give it a test

**Comment by omkarkhatavkar** [comment]
@sjha4 one quick test got failed here, Seems like conditional statement not working correctly. While saving any interval it shows the message "Please enter cron below" which does not looks correct. If you can make it as 'working' or 'saving' would be more correct.   

20190307154900-c33551585f gif-2-mp4 com

**Comment by omkarkhatavkar** [comment]
@sjha4 Looks Good now!

**Comment by jturel** [comment]
Looks good. Once the bastion change is merged please bump the gemspec here and we can merge this!

**Comment by sjha4** [comment]
> Looks good. Once the bastion change is merged please bump the gemspec here and we can merge this!

@jturel : Updated the gemspec with the new Bastion release..

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/sync-plans/details/sync-plan-details-info.controller.js [hunk 1]
```diff
@@ -41,7 +41,12 @@ angular.module('Bastion.sync-plans').controller('SyncPlanDetailsInfoController',
                 $scope.$watch('syncPlan.interval', function (interval) {
                     if (interval !== $scope.syncPlanInterval) {
                         $scope.editInterval = true;
-                        $scope.editedInterval = (interval === "custom cron");
+                        $scope.editedInterval = false;
+                        $scope.workingText = translate('Working');
+                        if (interval === "custom cron") {
+                            $scope.editedInterval = true;
+                            $scope.workingText = translate('Please enter cron below ');
```
Comment (jturel) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/sync-plans/details/sync-plan-details-info.controller.js]: What's the space after 'below' for?

Comment (sjha4) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/sync-plans/details/sync-plan-details-info.controller.js]: Updated..:+1:

**Pull request merged**